src/main/java/com/google/neighborgood/servlets/MessageServlet.java

```diff
@@ -131,6 +133,35 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     msgEntity.setProperty("sentTime", System.currentTimeMillis());
 
     datastore.put(msgEntity);
+
+    // After storing the message, create a notification entity to notify the receiver
```

Comment:
Maybe this notification code could be separated out into a function, e.g. `addNotificationToDataStore` or `notifyReceiver` or `notifyMessageReceiver`?

Reply:
Oh that would make the code looks better and more organized. I would separte the part as a function in the new commit.